backend/pyproject.toml

```diff
@@ -73,7 +73,7 @@ line-length = 120
 # Allow unused variables when underscore-prefixed.
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 
-# Assume Python 3.10.
+# TODO: Assume Python 3.10.
```

Comment:
@MelissaAutumn Do we need to update this? I did this PR on Python 3.12 with current versions of black and ruff...

Reply:
We can at least upgrade this to 3.11. I think 3.12 is fine, but 3.11 is the docker container's target.

Reply:
Alright, changed that. Thanks!